spec/game_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'simplecov'
+SimpleCov.start
+require './lib/stat_tracker'
+require './lib/game'
+
+RSpec.describe StatTracker do
+  before(:all) do
+    game_path = './data/games.csv'
+    team_path = './data/teams.csv'
+    game_teams_path = './data/game_teams.csv'
+
+    locations = {
+      games: game_path,
+      teams: team_path,
+      game_teams: game_teams_path
+    }
+
+    @stat_tracker = StatTracker.from_csv(locations)
+    # should we instantiate an object?
+  end
+
+  # it "exists" do
+  #   expect(@stat_tracker).to be_an_instance_of StatTracker
+  # end
+
+  it "#highest_total_score" do
+    expect(@stat_tracker.highest_total_score).to eq 11
+  end
+
+  it "#lowest_total_score" do
+    expect(@stat_tracker.lowest_total_score).to eq 0
+  end
+
+  it "#percentage_home_wins" do
+    expect(@stat_tracker.percentage_home_wins).to eq 0.44
+  end
+
+  it "#percentage_visitor_wins" do
+    expect(@stat_tracker.percentage_visitor_wins).to eq 0.36
+  end
+
+  it "#percentage_ties" do
+    expect(@stat_tracker.percentage_ties).to eq 0.20
+  end
+
+  it "#count_of_games_by_season" do
+    expected = {
```

Comment:
Let's use the spec harness to write and modify our test. I like how the expected outcome gives us a format to work with and helps us better determine what we are looking for in the CSV data. We mentioned this in our discussions and let's use this for our advantage with the key => value data structure.